codelabs/custom_generator/custom_generator.md

```diff
@@ -108,14 +114,27 @@ Blockly.common.defineBlocksWithJsonArray([{
 }]);
 ```
 
+This code creates the block definitions, but it doesn't register them with Blockly. We'll do that in `src/index.js`.
+Currently, we import `blocks` from the original sample file, `text.js`. Instead, we want to import the definitions we just added:
+
+```js
+import {blocks} from './blocks/json';
```

Comment:
Does this mean you deleted a line of code, or are you still keeping the text blocks around as well?

Reply:
It was more replacing the word `text` with the word `json`. But made this more explicit.